src/component/Nav/Nav.js

```diff
@@ -1,8 +1,91 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import Modal from './component/Modal';
 import './Nav.scss';
 
 const Nav = () => {
-  return <div>Nav</div>;
+  const [scrollPosition, setScrollPosition] = useState(0);
+
+  const updateScroll = () => {
+    setScrollPosition(window.scrollY || document.documentElement.scrollTop);
+  };
+
+  useEffect(() => {
+    window.addEventListener('scroll', updateScroll);
+  });
+
+  const [modalOpen, setModalOpen] = useState(false);
+  const openModal = () => {
+    setModalOpen(true);
+  };
+  const closeModal = () => {
+    setModalOpen(false);
+  };
+
+  return (
+    <nav className={`nav${scrollPosition < 5 ? '' : ' changeNav'}`}>
+      <div className="navCategory">
+        <ul className="navText">
+          <Link
+            to="/Survey"
+            className={scrollPosition < 5 ? '' : 'changeAtagText'}
+          >
+            Perfume Teller
+          </Link>
+        </ul>
+        <ul className="navText shop">
+          <div
+            className={scrollPosition < 5 ? '' : 'changeAtagText'}
+            onMouseOver={openModal}
+          >
+            SHOP
+          </div>
+          <Modal
+            open={modalOpen}
+            close={() => {
+              closeModal();
+            }}
```

Comment:
```suggestion
            modalOpen={modalOpen}
            closeModal={closeModal}
```

* props로 함수 전달 할 때는, 함수 자체를 전달 해 주시고, 이름을 같게 해주신는게 추적하기 용이합니다.